ee/cis/macos-14/test/scripts/CIS_3.3.sh

```
@@ -1,18 +1,29 @@
+    # Remove all_max=NNNM or all_max=NNNG patterns
+    { gsub(/all_max=[0-9]+[MG]/, "", $0) }
+    # On the "* file" line, ensure ttl=365 is present
+    /^\* file/ {
+        if ($0 ~ /ttl=[0-9]+/) {
+            gsub(/ttl=[0-9]+/, "ttl=365", $0)
+        } else {
+            $0 = $0 " ttl=365"
+        }
+    }
+    { print }
+' "$INSTALL_FILE" > "$TMP_FILE"
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)